bindata/manifests/config-units/initial-config-reset.yaml

```diff
@@ -0,0 +1,17 @@
+contents: |
```

Comment:
We don't need this systemd service anymore, do we?

Reply:
right! done